lib/notifications/block_waiver_backfiller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Notifications.BlockWaiverBackfiller do
+  use GenServer
+  import Ecto.Query
+  alias Notifications.Db.Notification, as: DbNotification
+  require Logger
+
+  @sleep_interval 5000
+
+  @batch_size 5
+
+  @spec default_name() :: GenServer.name()
+  def default_name(), do: Notifications.NotificationServer
+
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
+  def start_link(_opts \\ []) do
+    GenServer.start_link(__MODULE__, nil)
+  end
+
+  @impl true
+  def init(_) do
+    Process.send_after(self(), :backfill_batch, 0)
+    {:ok, nil}
+  end
+
+  @impl true
+  def handle_info(:backfill_batch, state) do
+    notifications_requiring_backfill_count =
+      Skate.Repo.one(
+        from(n in DbNotification, select: count(n), where: is_nil(n.block_waiver_id))
+      )
+
+    Logger.warn(
+      "backfill_batch notifications_requiring_backfill_count=#{
+        notifications_requiring_backfill_count
+      }"
+    )
```

Comment:
Could this maybe be `info` instead of `warn`? i worry about having too many `warn` messages to wade through when troubleshooting things.